applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.hpp

```diff
@@ -57,6 +58,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) SmallStrainUPwDiffOrderElement : pub
                                    std::unique_ptr<StressStatePolicy> pStressStatePolicy)
         : UPwBaseElement(NewId, pGeometry, pProperties, std::move(pStressStatePolicy))
     {
+        this->SetUpPressureGeometryPointer();
```

Comment:
See previous suggestion
```suggestion
        SetUpPressureGeometryPointer();
```